example/android/build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.8.0'
+    ext.kotlin_version = '1.6.10'
```

Comment:
any impact on our endusers due to this? 👀

Reply:
this is because the plugin is using 1.6.10, older versions of android studio won't work with 1.8.0